docs/code_ref/net.rst

```diff
@@ -1,92 +1,48 @@
+*************************
 Remote data (`sunpy.net`)
 *************************
 
-``sunpy.net`` contains a lot of different code for accessing various solar
-physics related web services. This submodule contains many layers. Most users
-should use `~sunpy.net.Fido`, which is an interface to multiple sources
-including all the sources implemented in `~sunpy.net.dataretriever` as well as
-`~sunpy.net.vso` and `~sunpy.net.jsoc`. `~sunpy.net.Fido` can be used like so::
+`sunpy.net` contains a lot of different classess for accessing various solar physics related web services.
 
-    >>> from sunpy.net import Fido, attrs as a
-    >>> results = Fido.search(a.Time("2012/1/1", "2012/1/2"), a.Instrument.lyra)  # doctest: +REMOTE_DATA
-    >>> files = Fido.fetch(results)  # doctest: +SKIP
+Most users should use `~sunpy.net.Fido`, which is an interface to multiple sources including all the sources implemented in `~sunpy.net.dataretriever` as well as `~sunpy.net.vso` and `~sunpy.net.jsoc`.
 
 .. automodapi:: sunpy.net
    :include-all-objects:
-   :headings: =-
-
-.. automodapi:: sunpy.net.attrs
-   :headings: ^"
 
 .. automodapi:: sunpy.net.fido_factory
-   :headings: ^"
 
-VSO
```

Comment:
I really don't think these headings should be removed.

Reply:
The current plan is to tidy up the python files docstrings for each module, see how they render in and then add back the headings.

Or maybe split up this page into several, its quite long.